### ISSUE TITLE
How to set the number of processes a.k.a. n_jobs?

### ISSUE THREAD

**Issue by FelixNeutatz** [awslabs/autogluon#1324]
Dear all,

I am using the TabularPredictor. However, I am missing a parameter to set the number of processes/jobs that are used.
Can somebody please give me hint how to configure that?

Thank you for your help.

Best regards,
Felix

**Comment by Innixma** [comment]
```

predictor = TabularPredictor(...).fit(..., ag_args_fit={'num_cpus': 16})

```

While not all models will respect this value currently, this will work for some models (the two neural networks).

If you happen to know the name of the hyperparameter that controls the cpu count for a particular algorithm, you can also pass it, such as:

```

hyperparameters={'GBM': {'n_jobs': 16}}

```

It's generally recommended to use the default (using all cores). We are currently working on parallel training via ray in the next release (v0.4).

**Comment by Innixma** [comment]
@ShaohongBai This is very interesting and not what I would expect. Is it possible for you to provide a minimally reproducible example script + data?

**Comment by ShaohongBai** [comment]
Understood, no more questions.

**Comment by knoam** [comment]
Hi @Innixma would you consider putting out another release with this fix before 0.4?

**Comment by Innixma** [comment]
@knoam The fix for the F1 metric issue / general speedups is in pre-release. You can get it via:

```
pip install --pre autogluon
```
